composer.json

```diff
@@ -130,7 +131,7 @@
         "phpstan-tests": "tools/phpstan analyze -c tests/phpstan.neon",
         "phpstan-baseline": "tools/phpstan --generate-baseline",
         "psalm-baseline": "tools/psalm  --set-baseline=psalm-baseline.xml",
-        "stan-setup": "phive install",
+        "stan-setup": "phive install && cp composer.json composer.backup && composer require --dev phpstan/extension-installer symplify/phpstan-rules && mv composer.backup composer.json",
```

Comment:
Couldn't these be standard dev deps? Having untracked changes sitting around isn't great.

Reply:
The could, if PHPStan would also already be IMO.
But as it is, phive is installing that one.
We could split it sure, not sure if that works, as those might be dependant on it.

Reply:
I wonder if phive would be able to install those, too.

Reply:
See also https://github.com/phar-io/phar.io/issues/142#issuecomment-1826863632